ewatercycle_experiment_launcher/api/pyewatercycle.py

```diff
@@ -0,0 +1,278 @@
+import textwrap
+from flask import current_app
+
+from nbformat import NotebookNode
+from nbformat.v4 import new_markdown_cell, new_code_cell, new_notebook
+
+from ewatercycle_experiment_launcher.generate import PY3_META
+from ewatercycle_experiment_launcher.process import process_notebook
+
+
+def notebook(setup: dict, forcing_root_dir: str) -> NotebookNode:
```

Comment:
Because it is set from command line, not from a request body. Also to make testing easier the notebook method is written without any flask globals.

Can be removed once https://github.com/eWaterCycle/ewatercycle/issues/248 is implemented.